src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -2110,21 +2110,24 @@ public void AarContentExtraction ([Values (false, true)] bool useAapt2)
 			}
 		}
 
+		const   string  FirstFrameworkVersion           = "v4.4";
+		const   string  AlternateFrameworkVersion       = "v5.0";
+
```

Comment:
`Builder` has a method to grab the "latest" framework version:
https://github.com/xamarin/xamarin-android/blob/79b220a05f643b72608f597de3cf4a70b05b85e1/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs#L175-L202

Does it need a version to grab the oldest? So when we drop more these tests won't break?

Reply:
Sounds like a sensible way to do things.